Lib/multiprocessing/util.py

```diff
@@ -102,6 +102,27 @@ def log_to_stderr(level=None):
     _log_to_stderr = True
     return _logger
 
+
+# Abstract socket support
+
+def _platform_supports_abstract_sockets():
+    if "linux" in sys.platform:
+        return True
+    if hasattr(sys, 'getandroidapilevel'):
```

Comment:
nitpick: Currently, sys.platform == "linux" on Android. But it's fine to keep this code path.